proto/eraftpb.proto

```diff
@@ -72,11 +72,13 @@ message HardState {
 
 message ConfState {
     repeated uint64 nodes = 1;
+    repeated uint64 learner_nodes = 2;
```

Comment:
Why not `learners`[0]?

[0] https://github.com/coreos/etcd/pull/8751/files#diff-ed53c23ad570cdd6dce39cbd1b751cacR80